.travis.yml

```
@@ -15,6 +15,8 @@ matrix:
       dist: xenial
     - python: "3.8"
       dist: xenial
+    - python: "3.9-dev"
```

```suggestion
    - python: "3.9"
```

3.9 is now available on Travis, but it may be worth ditching Travis for GHA:

* https://www.jeffgeerling.com/blog/2020/travis-cis-new-pricing-plan-threw-wrench-my-open-source-works